### PR TITLE
added data processing methods

### PR DESCRIPTION
**Pull request by tsehelnyk**

### CODE REVIEW

File: src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java [hunk 1]
```diff
@@ -12,38 +18,44 @@
...
+    public Set<String> getMostPopularWordsInReviews() {
+        Map<String, Integer> words = reviewRepository.getReviewsText()
+                .stream()
+                .flatMap(text -> Arrays.stream(text.replaceAll("[^a-zA-Z\\s]", "").split(" ")))
+                .collect(Collectors.toMap(k -> k, k -> 1, (a, b) -> a + b, HashMap::new));
+
+        return words.values()
+                .stream()
+                .distinct()
+                .flatMap(val -> getKeys(words, val).stream())
```
Comment (Sofasmile) [inline src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java:54]: do we need to sort words?

Comment (tsehelnyk) [reply src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java:54]: we sort not words, but their frequency

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java (hunk 1 above)
Comment (tsehelnyk) [reply src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java:54]: this is mistake(
distinct remove repeated values...

Comment (tsehelnyk) [reply src/main/java/com/dev/boot/amazonreviewsapp/service/impl/ReviewServiceImpl.java:54]: fixed!

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**